app/AppGroup.m

```diff
@@ -41,14 +41,14 @@
         return nil;
     if (dl_info.dli_fbase == NULL)
         return nil;
-    uint8_t *base = dl_info.dli_fbase;
+    char *base = dl_info.dli_fbase;
```

Comment:
What does this fix?

Reply:
`char *` and `void *` are the legal aliasing types

Reply:
Guilty as charged

Reply:
I should note that *any* character type is a legal aliasing type, so you are also welcome to use `signed char *` and `unsigned char *` if you so wish.